view/frontend/web/js/view/payment/method-renderer/rvvup-method.js

```diff
@@ -162,6 +181,120 @@ define([
                 return window.checkoutConfig.payment[this.index].style;
             },
 
+            renderCardForm: function () {
+
+                if (rvvup_parameters.settings.card.flow === "INLINE") {
+                    $('body').trigger("processStart");
+                    window.rendered = false;
```

Comment:
Can we change this to window.rvvup_card_rendered to avoid any future conflicts with other libraries/plugins and ourselves?